src/main/resources/org/jvnet/hudson/plugins/SSHBuildWrapper/global.jelly

```diff
@@ -7,14 +7,14 @@
       <f:repeatable var="site" name="sites" items="${descriptor.sites}">
         <table width="100%">
           <f:entry title="${%Hostname}" help="/plugin/ssh/help-hostname.html">
-            <f:textbox name="hostname" value="${site.hostname}" checkUrl="'${rootURL}/descriptorByName/org.jvnet.hudson.plugins.SSHBuildWrapper/checkHostname?hostname='+escape(this.value)" />
+            <f:textbox name="hostname" value="${site.hostname}" field="hostname" />
```

Comment:
I think this could be simplified further, if you're willing to do so. Because adding `field` attribute creates data binding you don't need `name` and `value` attributes here anymore. Also I find adding `field` to the `<f:entry>` instead a bit cleaner, but might just be my preference.
Same applies to below.

Reply:
I agree this should be simplified further if practical.

Reply:
Great point. Made those changes